### PR TITLE
Receiving multipart form data from firecloud-ui to create entities from JSON

### PR DESCRIPTION
**Pull request by helgridly**
The first part of [DSDEEPB-814](https://broadinstitute.atlassian.net/browse/DSDEEPB-814).

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala [hunk 1]
```diff
@@ -4,12 +4,15 @@ import java.text.SimpleDateFormat
 import java.util.Date
 
 import akka.actor.{Actor, Props}
-import com.wordnik.swagger.annotations.{Api, ApiImplicitParam, ApiImplicitParams, ApiOperation}
+import com.wordnik.swagger.annotations._
+import org.broadinstitute.dsde.firecloud.model.{EntityCreateResult}
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
```
Comment (dmohs) [inline src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala]: Our team's style guide [1] specifies an import order...
[1] https://broadinstitute.atlassian.net/wiki/display/DSDE/Code+Style+Conventions

File: src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala [hunk 2]
```diff
@@ -85,4 +88,25 @@ trait WorkspaceService extends HttpService with FireCloudDirectives {
...
+    notes = "Create entities from a list of JSON objects. This won't be the final API.")
+  @ApiResponses(Array(
+    new ApiResponse(code = 200, message = "Successful"),
+    new ApiResponse(code = 500, message = "Internal Error")))
+  def importEntitiesRoute: Route =
+    path(ApiPrefix / Segment / Segment / "importEntitiesJSON" ) { (workspaceNamespace, workspaceName) =>
+      post {
+        formFields( 'entities ) { (entities) =>
+          respondWithJSON { requestContext =>
+            //TODO: Parse the entities string as JSON, fire off create requests to Rawls, and combine the responses in the EntityCreateResult case class.
```
Comment (dmohs) [inline src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala]: ... and a column limit for lines (100).

**Comment by dmohs** [comment]
Looks good to me (we can fix the code style), but will you take a look @rushtong?

**Comment by rushtong** [comment]
I'd like to see a mock test added for this. With that in place, the tests should *just work* when the todo is taken care of.

Outside of that, it looks good.

**Comment by rushtong** [comment]
@helgridly - please rebase off of master and I'll handle the merge. Thank you!

**Comment by helgridly** [comment]
I'll fix the code style issues too.

**Pull request merged**